packaging/pkg_helpers.bash

```diff
@@ -276,3 +276,15 @@ setup_junit_results_folder() {
     export CONDA_PYTORCH_BUILD_RESULTS_DIRECTORY="${SOURCE_ROOT_DIR}/build_results/results.xml"
   fi
 }
+
+install_onnx_runtime_on_linux() { 
+  if [[ "$(uname)" == Linux ]] ; then
+    case "$PYTHON_VERSION" in
+      # TODO: Enable when onnxruntime is packaged for python 3.8
+      3.8) ;;
+      *)
+        pip install onnxruntime 
```

Comment:
would we want to use `pip_install` instead? It uses a retry and removes the progress bar

Reply:
Thanks, I will do.